requirements.txt

```diff
@@ -9,3 +9,4 @@ xformers >= 0.0.19
 fastapi
 uvicorn
 pydantic  # Required for OpenAI server.
+fschat  # Required for OpenAI ChatCompletion Endpoint.
```

Comment:
Doesn't this lead to a circular dependency problem? While FastChat does not have vLLM as its dependency, it looks a bit weird and I'm not sure this would not cause any problem...

Reply:
Yeah I agree this is weird... We only use one file from FastChat. Maybe we can avoid this potential circular dependency problem by only downloading that one file. Let's keep it for now since it works now.